v3/src/components/graph/components/attribute-label.tsx

```diff
@@ -16,9 +18,11 @@ interface IAttributeLabelProps {
 export const AttributeLabel = forwardRef<SVGGElement, IAttributeLabelProps>(
   ({ transform, attributeIDs }:IAttributeLabelProps, ref) => {
   const dataConfiguration = useDataConfigurationContext(),
-    attrNames = attributeIDs.map(anID => dataConfiguration?.dataset?.attrFromID(anID).name),
+    dataset = dataConfiguration?.dataset,
+    attrNames = attributeIDs.map(anID => dataset?.attrFromID(anID).name),
     svgRef = ref as MutableRefObject<SVGGElement | null>,
-    labelRef = useRef<any>()
+    labelRef = useRef<any>(),
+    [, setCounter] = useState(0)
```

Comment:
FYI, CLUE has a `useForceUpdate()` hook that encapsulates this trope.

Reply:
But it's not currently part of our codebase, right?